### PR TITLE
Render mouseover tooltip for the new tables

### PR DESCRIPTION
**Pull request by Georgi2704**

### CODE REVIEW

File: packages/orchestrator-ui-components/src/components/WfoTable/WfoTable/WfoDataCell.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React, { ReactNode } from 'react';
+
+import { EuiToolTip } from '@elastic/eui';
+
+interface WfoDataCellProps {
+    showTooltip: boolean;
+    content: ReactNode;
```
Comment (ricardovdheijden) [inline packages/orchestrator-ui-components/src/components/WfoTable/WfoTable/WfoDataCell.tsx:7]: If i understand this correct this is the content inside the tooltip ietself and not the actual data in the cell?
Therefore I would suggeste to rename this into `tooltipContent`

**Review by ricardovdheijden — APPROVED** [review]

File: packages/orchestrator-ui-components/src/components/WfoBadges/WfoBadge/WfoBadge.tsx [hunk 2]
```diff
@@ -18,7 +18,7 @@ export const WfoBadge: FC<WfoBadgeProps> = ({
     ...restProps
```
Comment (ricardovdheijden) [inline packages/orchestrator-ui-components/src/components/WfoBadges/WfoBadge/WfoBadge.tsx:18]: don't forget the changeset

File: packages/orchestrator-ui-components/src/components/WfoBadges/WfoBadge/WfoBadge.tsx [hunk 3]
```diff
@@ -18,7 +18,7 @@ export const WfoBadge: FC<WfoBadgeProps> = ({
...
 }) => (
     <EuiBadge {...restProps}>
-        <EuiText color={textColor} size={size ?? 'xs'}>
+        <EuiText title="" color={textColor} size={size ?? 'xs'}>
```
Comment (DutchBen) [inline packages/orchestrator-ui-components/src/components/WfoBadges/WfoBadge/WfoBadge.tsx:21]: What does the empty title prop? 
```suggestion
        <EuiText color={textColor} size={size ?? 'xs'}>
```

File: packages/orchestrator-ui-components/src/components/WfoTable/WfoTable/WfoDataCell.tsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+import { EuiToolTip } from '@elastic/eui';
+
+import { useOrchestratorTheme } from '@/hooks';
+
+interface WfoDataCellProps {
+    tooltipContent: ReactNode;
+    children: ReactNode;
+}
+
+export const WfoDataCell = ({ tooltipContent, children }: WfoDataCellProps) => {
```
Comment (DutchBen) [inline packages/orchestrator-ui-components/src/components/WfoTable/WfoTable/WfoDataCell.tsx:12]: I think we said - againt my preference :) - we'd use FC here

**Review by DutchBen — APPROVED** [review]

File: packages/orchestrator-ui-components/src/components/WfoBadges/WfoBadge/WfoBadge.tsx (hunk 3 above)
Comment (Georgi2704) [reply packages/orchestrator-ui-components/src/components/WfoBadges/WfoBadge/WfoBadge.tsx:21]: It prevents showing the default browser tooltip. We don't want those showing.

File: packages/orchestrator-ui-components/src/icons/WfoCheckmarkCircleFill.tsx [hunk 5]
```diff
@@ -14,7 +14,6 @@ export const WfoCheckmarkCircleFill: FC<WfoIconProps> = ({
         version="1.1"
         xmlns="http://www.w3.org/2000/svg"
     >
-        <title>icon/checkmark-circle-fill</title>
```
Comment (ricardovdheijden) [inline packages/orchestrator-ui-components/src/icons/WfoCheckmarkCircleFill.tsx:17]: Adjusting SVG files might not be the best solution. Can't you apply a css rule?

**Review by ricardovdheijden — APPROVED** [review]

File: packages/orchestrator-ui-components/src/icons/WfoCheckmarkCircleFill.tsx (hunk 5 above)
Comment (Georgi2704) [reply packages/orchestrator-ui-components/src/icons/WfoCheckmarkCircleFill.tsx:17]: I don't think it is possible only with CSS. I've tried before but didn't succeed. 

And if there is a way with css, we always have to hide the default tooltip (title). Isn't it better to just remove it from the svg ?

**Pull request merged**